engines/sword25/gfx/graphicengine.h

```diff
@@ -231,8 +231,8 @@ class GraphicEngine : public ResourceService, public Persistable {
 	// -------------------
 	bool persist(OutputPersistenceBlock &writer) override;
 	bool unpersist(InputPersistenceBlock &reader) override;
-
-	static void ARGBColorToLuaColor(lua_State *L, uint color);
+    bool isRTL();
+    static void ARGBColorToLuaColor(lua_State *L, uint color);
```

Comment:
For whatever reason, you changed the indentation here from tabs to spaces. Please revert.

Also, please fix the previous line too.

Reply:
Done